LeaderboardBackend.Test/Controllers/LeaderboardsControllerTest.cs

```diff
@@ -30,12 +28,12 @@ public void Setup()
 	public async Task GetLeaderboard_NotFound_LeaderboardDoesNotExist()
 	{
 		_leaderboardServiceMock
-			.Setup(x => x.GetLeaderboard(It.IsAny<long>()))
+			.Setup(x => x.GetLeaderboard(It.IsAny<ulong>()))
 			.Returns(Task.FromResult<Leaderboard?>(null));
 
-		ActionResult<Leaderboard> response = await _controller.GetLeaderboard((long)1);
+		var response = await _controller.GetLeaderboard(1);
```

Comment:
This is a scenario where I wouldn't use var. It's not obvious from reading the line what the type is going to be, so declaring that this is an `ActionResult<Leaderboard>` is useful information.

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`LeaderboardsControllerTest.cs` [`R32`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-b000a0cefe2a75042e27301251d7042a9a1d4ac64a79f5573cbcae6a29999fb8R67)).